package-scripts.js

```diff
@@ -29,9 +29,9 @@ module.exports = {
         fix: 'eslint --fix'
       },
       react: {
-        default: crossEnv('BABEL_TARGET=node jest --no-cache --config jest.React.json --notify'),
-        accept: crossEnv('BABEL_TARGET=node jest -u --no-cache --config jest.React.json --notify'),
-        watch: crossEnv('BABEL_TARGET=node jest --watch --no-cache --config jest.React.json --notify')
+        default: crossEnv('BABEL_TARGET=node jest --no-cache --config jest.React.json --notify --updateSnapshot'),
+        accept: crossEnv('BABEL_TARGET=node jest -u --no-cache --config jest.React.json --notify --updateSnapshot'),
```

Comment:
@DannyMcwaves the update snapshot should only be 'accept' please fix this, thanks